package.js

```diff
@@ -28,7 +30,9 @@ Package.on_use(function(api) {
     api.imply([
         "accounts-base",
         "softwarerero:accounts-t9n@1.0.6",
-        "iron:router@1.0.7",
+        "meteorhacks:flow-router@1.1.1",
+        "meteorhacks:flow-layout@1.0.1",
```

Comment:
try updating flow layout to `1.0.2`. We fixed a major bug which affects when switching between layouts.